dask/tests/test_distributed.py

```diff
@@ -343,6 +359,10 @@ def test_blockwise_dataframe_io(c, tmpdir, io, fuse, from_futures):
     pd = pytest.importorskip("pandas")
     dd = pytest.importorskip("dask.dataframe")
 
+    # TODO: this configuration is flaky on osx in CI
```

Comment:
Optional, but linking back to the corresponding issue is handy 

```suggestion
    # TODO: this configuration is flaky on osx in CI
    # See https://github.com/dask/dask/issues/8816
```